web/packages/teleterm/src/mainProcess/createAgentConfigFile.ts

```diff
@@ -67,8 +67,46 @@ export async function createAgentConfigFile(
       timeout: 10_000, // 10 seconds
     }
   );
+
+  try {
+    await fs.mkdir(path.dirname(configFile), {
+      // Directories must be rwx, not just rw-.
+      mode: 0o744,
```

Comment:
I spelled out modes for created dirs and files explicitly, but it's not necessarily needed. I got scared when I realized that Node.js by default creates files and folders with 0o777 (readable, writeable and executable by everyone). It doesn't matter as long as we create files within folders managed by Electron, since those have 0o700 set, so no one is able to traverse them but the current user.

AFAIK the only place where we create a dir outside of an Electron-managed directory is in Connect My Computer when we download an archive with Teleport binaries to a temp folder. `fs.mkdtemp` sets `0o700` fortunately.

Reply:
So it seems that specifying the modes here don't give us much. Maybe we should we remove them?